docs/contributing/tests.md

```diff
@@ -29,3 +29,25 @@ samples are provided. To run the tests, follow this procedure:
   `tests/secrets.sample.yaml`).
 
 * Run `make test-minimal`.
+
+
+## Running tests on systems where /bin/sh is not /bin/bash
```

Comment:
I'm not sure we need this text now that it's properly defaulted? If something is not necessary to know to set the test suite up, i wouldn't put it here, to avoid "reading fatigue". It's not a blocker but IMO this may get removed in a test suite refactoring later.